### PR TITLE
[oree-842] Adding support for webhook to manifest

### PR DESCRIPTION
**Pull request by nimal-outreach**
This PR is defining webhooks-related application attributes in manifest,  SDK, and in OREX service schema 

`app {
  webhooks: {
      events: ["*"],
      url: "https://application-host.com/webhook"
  }
}`

### CODE REVIEW

**Comment by nimal-outreach** [comment]
@antonaudzei-outreach can you please verify if I've updated properly the JSON schema?

**Review by sheff146 — APPROVED** [review]

File: src/manifest/Application.ts [hunk 1]
```diff
@@ -62,28 +63,27 @@ export class Application {
...
+   * @see https://github.com/getoutreach/extensibility-sdk/blob/master/docs/manifest.md#webhooks
+   * @see https://github.com/getoutreach/extensibility-sdk/blob/master/docs/webhooks.md
    *
-   * @see https://github.com/getoutreach/extensibility-sdk/blob/main/docs/url-parsing.md#timeout-handling
-   * @type {boolean}
+   * @type {ManifestWebhook}
    * @memberof Application
    */
-  public disableTimeoutMonitoring?: boolean;
+  public webhooks?: ManifestWebhook;
```
Comment (palootcenas-outreach) [inline src/manifest/Application.ts:88]: The plural name applies that we support multiple webhooks, but it is actually a single webhook with a single URL.

Did you consider using singular `webhook` name here or allowing multiple webhooks (thus allowing calling multiple URLs) ?

**Review by palootcenas-outreach — APPROVED** [review]

File: src/manifest/Application.ts (hunk 1 above)
Comment (nimal-outreach) [reply src/manifest/Application.ts:88]: Asking Anton about this and I'll rename if it is only one

Comment (nimal-outreach) [reply src/manifest/Application.ts]: renamed

**Pull request merged**